### PR TITLE
chore: remove DisablePow from PersistentFlags

### PR DESCRIPTION
**Pull request by logan-wrld**
This fix is related to issue #261 - Gorka brought up not having the pow-signal set as a PersistentFlag. The cmd does not shouldn't need to be accessible to the other services as described here: https://github.com/Lilypad-Tech/lilypad/pull/261#discussion_r1695150837. 

I have removed the --disable-pow from the persistent flags and recorded a video of the disable-pow true/false value being displayed in a print statement. 

https://github.com/user-attachments/assets/455a94a0-9503-461e-9927-2e9924e6820f

- To startup the RP with POW on, you use `./stack resource-provider`
- To startup the RP with the POW off, you use `./stack resource-provider --disable-pow`

### Review Type Requested (choose one):
- [ ] **Glance** - superficial check (from domain experts)
- [ ] **Logic** - thorough check (from everybody doing review)

### Summary
Provide a one line summary and link to any relevant references

### Task/Issue reference

Closes: add_link_here

### Details (optional)
There was a previous PR here where the parsing cmd was added to fix the pow flag - making it work correctly

Previous PR modifications: https://github.com/Lilypad-Tech/lilypad/pull/261/commits/380f590ed9ff4204144feb9be791dbffd31ca9e4

Original Issue Here: https://github.com/Lilypad-Tech/hni/issues/21

### How to test this code? (optional)
Startup the services and run `./stack resource-provider --disable-pow` to run the RP without POW being executed.

### CODE REVIEW

**Review by AquiGorka — APPROVED** [review]
Thank you @noryev for the change here.

Before you merge:
I don't think this is a fix as it does not introduce nor change user facing features (more of a `chore`, @bgins is our in-house-expert for labelling changes so I refer to him when in doubt).

And [nit] let's make sure PR branches follow the schema of `name/type-something-something` (`logan/chore-remove-unnecessary-root-flag`) - no need to change the branch, just a reminder for any next line of work.

**Review by walkah — APPROVED** [review]
Good catch, Gorka! Thank you 🙏 I need to remember that root here is *all services* 🤦

**Comment by bgins** [comment]
> Before you merge: I don't think this is a fix as it does not introduce nor change user facing features (more of a `chore`, @bgins is our in-house-expert for labelling changes so I refer to him when in doubt).

Some gray area on this one because services other than resource providers will have had the `--pow-signal` available in help output, though it wouldn't have done anything.

But I would like to keep it as a `fix` because we are almost ready to introduce a labeling change that interacts with the release pipeline (#260), and I would like to merge this immediately after to keep an eye on the release.

**Pull request merged**
